src/dcp/dcp.c

```diff
@@ -868,6 +870,12 @@ int main(int argc, char** argv)
             case 'P':
                 mfu_progress_timeout = atoi(optarg);
                 break;
+            case 'X':
+                mfu_copy_opts->preserve_xattrs = true;
+                if(rank == 0) {
+                    MFU_LOG(MFU_LOG_INFO, "Preserving extended attributes.");
+                }
+                break;
```

Comment:
Since the short option `X` for `--daos-prefix` isn't user-facing, it's okay to reuse `X` for something else. But I do think we need to change `--daos-prefix` to use _**some**_ other character (it doesn't really matter). Otherwise, with DAOS support enabled these options will clash.